ambry-api/src/main/java/com/github/ambry/quota/StorageUsageRefresher.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+import java.util.Map;
+
+
+/**
+ * An interface to initialize and refresh the storage usage for each container.
```

Comment:
Sorry about the nitpick again, but I am not sure what do we mean by refresh here? Does it mean refreshing a cache, or a backend. Or does it mean updating something with new information?

Reply:
It means if the storage usage is changed, by any other outsider, the implementation should be able to re-fetch the storage usage and refresh the in memory cache.